spec/core/data_structures.md

```diff
@@ -455,8 +466,8 @@ func SumTruncated(bz []byte) []byte {
 
 | Name          | Type   | Description                   | Field Number |
 |---------------|--------|-------------------------------|--------------|
-| message_delay | [google.protobuf.Duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) | Bound for how long a proposal message may take to reach all validators on a newtork and still be considered valid. | 1            |
-| precision     | [google.protobuf.Duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) | Bound for how skewed a proposer's clock may be from any validator on the network while still producing valid proposals. | 2            |
+| precision     | [google.protobuf.Duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) | Bound for how skewed a proposer's clock may be from any validator on the network while still producing valid proposals. | 1            |
```

Comment:
Skew is between clocks, correct? Between the proposer's clock and the clocks of any validator.

Reply:
Yes that is correct. Maybe it might be better to use the language "This sets a bound on how skewed a proposer's clock may be from any validator on the network while still producing valid proposals.". Do you think i should link to [proposer-based-timestamps](https://github.com/tendermint/tendermint/tree/331860c2a8c406fa89575165322b963925fb38c5/spec/consensus/proposer-based-timestamp) to be more clear?

Reply:
Yes, this is a good idea. I would just remove "This sets a" from the sentence.